tests/distributed/test_pipeline_parallel.py

```diff
@@ -18,22 +18,23 @@
 VLLM_MULTI_NODE = os.getenv("VLLM_MULTI_NODE", "0") == "1"
 
 
-@pytest.mark.parametrize(("TP_SIZE, PP_SIZE, EAGER_MODE, CHUNKED_PREFILL, "
+@pytest.mark.parametrize(("TP_SIZE, PP_SIZE, EAGER_MODE, CHUNKED_PREFILL, TRUST_REMOTE_CODE "
```

Comment:
@Manikandan-Thangaraj-ZS0321 Seems that a comma is missing here: `TRUST_REMOTE_CODE ` -> `TRUST_REMOTE_CODE, `

```
=============================================================================== ERRORS ===============================================================================
____________________________________________________ ERROR collecting tests/distributed/test_pipeline_parallel.py ____________________________________________________
tests/distributed/test_pipeline_parallel.py::test_compare_tp: in "parametrize" the number of names (6):
  ['TP_SIZE', 'PP_SIZE', 'EAGER_MODE', 'CHUNKED_PREFILL', 'TRUST_REMOTE_CODE MODEL_NAME', 'DIST_BACKEND']
must be equal to the number of values (7):
  (1, 2, 1, 1, 1, 'OpenGVLab/InternVL2-8B', 'ray')
====================================================================== short test summary info =======================================================================
ERROR tests/distributed/test_pipeline_parallel.py - Failed: tests/distributed/test_pipeline_parallel.py::test_compare_tp: in "parametrize" the number of names (6):
!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!! Interrupted: 1 error during collection !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
========================================================================== 1 error in 0.36s ==========================================================================
```

Reply:
Done